Tasks/UseDotNetV2/globaljsonfetcher.ts

```diff
@@ -64,6 +65,16 @@ export class globalJsonFetcher {
 
 }
 
-class GlobalJson {
-    public sdk: { version: string };
+export class GlobalJson {    
+    constructor(version: string | null = null) {        
+        if(version != null){
+            this.sdk = new sdk();
+            this.sdk.version = version;
+        }
+    }
+    public sdk: sdk;
+}
+
+class sdk{
```

Comment:
Do we need this class ?

Reply:
See the comment [here](https://github.com/microsoft/azure-pipelines-tasks/pull/10734#discussion_r302825241)